python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -173,6 +217,10 @@ def evaluate(
                 self.logger.info(
                     "Finished evaluating {} batches".format(batch_count))
 
+        # performing click rank distribution t-test
+        if ttest_pvalue_threshold > 0:
+            self.run_ttest(agg_mean, (agg_M2 / (agg_count - 1)), agg_count, ttest_pvalue_threshold, metrics_dict)
```

Comment:
@mohazahran This is not a good coding practice - passing a dictionary into a function which modifies it in place. A cleaner way to do this is to return `t_test_metrics_dict` from `run_ttest` and then do a `metrics_dict = metrics_dict.update(...)` here. That way someone reading this function can already follow all the values that metrics_dict contains. Thoughts?